spec/models/article_spec.rb

```diff
@@ -14,11 +17,12 @@
     it { is_expected.to validate_presence_of(:body) }
     it { is_expected.to validate_presence_of(:image) }
     it { is_expected.to validate_presence_of(:written_by) }
+    it { is_expected.to validate_presence_of(:category_id) }
   end
-
+  
```

Comment:
You need to test the association between article and category here

Reply:
done